src/strategies/masterchef-pool-balance/index.ts

```diff
@@ -124,7 +127,7 @@ function processValues(values: any[], options: any): number {
     result = poolStaked.mul(weight).div(weightDecimals);
   } else {
     const uniTotalSupply = values[1][0];
-    const uniReserve = values[2][0];
+    const uniReserve = options.uniPairToken0 ? values[2][0] : values[2][1];
```

Comment:
Hi @narwhalosaurus I feel like this may affect existing spaces using it (18 spaces) because not all spaces pass  `uniPairToken0` right now, can you plz look for a workaround, thanks